test/typeclass/traversable_test.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:test/test.dart';
+import 'package:fpdart/fpdart.dart';
+import 'package:fpdart/src/typeclass/traversable.dart';
+
```

Comment:
I realise this is probably not the most appropriate place for this code / tests yet but I just stuck it here to begin with and get the ball rollin